ArmPkg/Library/PlatformBootManagerLib/PlatformBm.c

```diff
@@ -1181,3 +1181,56 @@ PlatformBootManagerUnableToBoot (
     EfiBootManagerBoot (&BootManagerMenu);
   }
 }
+
+// MU_CHANGE Starts: Project Mu BDS has this function
+
+/**
+  BDS Entry  - DXE phase complete, BDS Entered.
+*/
+VOID
+EFIAPI
+PlatformBootManagerBdsEntry (
+  VOID
+  )
+{
+  return;
+}
+
+/**
+  HardKeyBoot
+*/
+VOID
+EFIAPI
+PlatformBootManagerPriorityBoot (
+  UINT16  **BootNext
+  )
+{
+  return;
+}
+
+/**
+  This is called from BDS right before going into front page
+  when no bootable devices/options found
+**/
+VOID
+EFIAPI
+PlatformBootManagerProcessBootCompletion (
+  IN EFI_BOOT_MANAGER_LOAD_OPTION  *BootOption
+  )
+{
+  return;
+}
+
+/**
+  OnDemandConInCOnnect

```

Comment:
We should do a better job actually describing the function in the comment. You don't have to do that here though.

Reply:
Agreed. We need to update that definition from the MdeModulePkg in the first place.